src/python/include/Persistent_cohomology_interface.h

```diff
@@ -73,6 +72,43 @@ persistent_cohomology::Persistent_cohomology<FilteredComplex, persistent_cohomol
     return persistence;
   }
 
+  std::vector<std::vector<int>> cofaces_of_cubical_persistence_pairs() {
+
+    // Warning: this function is meant to be used with CubicalComplex only!!
+
+    auto pairs = persistent_cohomology::Persistent_cohomology<FilteredComplex,
+      persistent_cohomology::Field_Zp>::get_persistent_pairs();
+
+    // Gather all top-dimensional cells and store their simplex handles
+    std::vector<int> max_splx; for (auto splx : stptr_->top_dimensional_cells_range()){
+      max_splx.push_back(splx); 
+    }
+    // Sort these simplex handles and compute the ordering function
+    // This function allows to go directly from the simplex handle to the position of the corresponding top-dimensional cell in the input data 
+    std::map<int, int> order; std::sort(max_splx.begin(), max_splx.end()); 
```

Comment:
Don't they come already sorted? I am not completely sure because the doc says in lexicographic order, but I think that should match.

Reply:
Indeed. I commented the sort.